prac_07/programming_language.py

```diff
@@ -39,4 +40,4 @@ def run_tests():
 
 
```

Comment:
The requirement is as long as you add your pointer arithmetic value at the string and run the test function, it is just ok.